src/gebt_poc/clamped_beam.h

```diff
@@ -25,7 +25,7 @@ class ClampedBeamLinearizationParameters : public LinearizationParameters {
     /// Define a static beam element with the given position vector for the nodes, 6x6
     /// stiffness matrix, and a quadrature rule
     ClampedBeamLinearizationParameters(
-        Kokkos::View<double*> position_vectors, StiffnessMatrix stiffness_matrix,
+        Kokkos::View<double**> position_vectors, StiffnessMatrix stiffness_matrix,
```

Comment:
Should these views have a defined size on the known dimension?

```suggestion
        Kokkos::View<double*[7]> position_vectors, StiffnessMatrix stiffness_matrix,
```

Reply:
Yes